### DISCUSSION TITLE
How to run widevine in linux mint 20.1 in qutebrowser

### DISCUSSION THREAD

**Discussion by logicalbhanu** [qutebrowser/qutebrowser#6293] [Q&A] [answered]
To play drm content we need widevine plugin in qutebrowser, there are some tutorial online regarding how to extract widevine plugin from chrome and put it in some qutebrowser folder(don't know which) to play drm content.

The problem is none of them work with me, I have my qutebrowser installed in a virtual environment on my linux mint 20.1 and I am also able to extract libwidevine.so from chrome appropriate version (depending upon version of qtwebengine of my qutebrowser), but don't know where I should put that file and what to do further for running drm content on my qutebrowser, can anyone guide me on this.

**Comment by The-Compiler** [comment, marked as answer]
Like I've said in https://github.com/qutebrowser/qutebrowser/issues/5388#issuecomment-799390280:

> If you have Qt installed in a virtualenv, you don't have any proprietary codec support, so I doubt Widevine will work.

**Reply by logicalbhanu** [reply]
thanks for the reply, I change my installation and install qutebrowser in my system, can you guide me now, how to solve the widevine issue.

**Reply by The-Compiler** [reply]
Not really, as this is likely distribution specific and I don't have a Linux Mint ready. However, with that setup, the tutorials you had mentioned might now work. If you find a working setup, it'd be great if you could document it so it can be added to the docs.

For a rough guide, see https://www.reddit.com/r/qutebrowser/comments/jwb90w/how_do_i_get_drm_content_netflix_spotify_to_work/gcprrt2?utm_source=share&utm_medium=web2x&context=3

**Comment by tom-i** [comment]
@logicalbhanu - did you solve this in Linux Mint?
